_search-plugins/ubi/ubi-dashboard-tutorial.md

```diff
@@ -0,0 +1,94 @@
+
+OpenSearch Dashboards prompts you to filter on any `date` field in your schema so that you can look at things like trending queries over the last 15 minutes. However, for your first dashboard, select **I don't want to use the time filter**, as shown in the following image. 
+<img src="{{site.url}}{{site.baseurl}}/images/ubi/index_pattern3.png" alt="Index Patterns" width="400"/>
+
+
+After selecting **Create index pattern**, you're ready to start building a dashboard that displays the UBI store data.
+
+## 3. Create a new dashboard
+
+To create a new dashboard, on the top menu, select **OpenSearch Dashboards > Dashboards** and then **Create > Dashboard** > **Create new**.
+If you haven't previously created a dashboard, you are presented with the option to create a new dashboard. Otherwise, previously created dashboards are displayed.
+
+
+In the **New Visualization** window, select **Pie** to create a new pie chart. Then select the index pattern you created in step 2.
+
+Most visualizations require some sort of aggregate function on a bucket/facet/aggregatable field (numeric or keyword). You'll add a `Terms` aggregation to the `action_name` field so that you can view the distribution of event names. Change the **Size** to the number of slices you want to display, as shown in the following image.
+![Pie Chart]({{site.url}}{{site.baseurl}}/images/ubi/pie.png "Pie Chart")
+
+Save the visualization so that it's added to your new dashboard. Now that you have a visualization displayed on your dashboard, you can save the dashboard.
+
+## 4. Add a tag cloud visualization
+
+Now you'll add a word cloud for trending searches by creating a new visualization, similarly to the previous step.  
+
+In the **New Visualization** window, select **Tag Cloud**, and then select the index pattern you created in step 2. Choose the tag cloud visualization of the terms in the `message` field where the JavaScript client logs the raw search text. Note: The true query, as processed by OpenSearch with filters, boosting, and so on, resides in the `ubi_queries` index. However, you'll view the `message` field of the `ubi_events` index, where the JavaScript client captures the text that the user actually typed. 
+
+The following image shows the tag cloud visualization on the `message` field.
+![Word Cloud]({{site.url}}{{site.baseurl}}/images/ubi/tag_cloud1.png "Word Cloud")
+
+The underlying queries can be found at [SQL trending queries]({{site.url}}{{site.baseurl}}/search-plugins/ubi/sql-queries/#trending-queries).
+{: .note} 
+
+
+The resulting visualization may not be exactly what you're looking for. The `message` field is updated with every ---not only query/search s---and can be used in any way desired. As a result, it can contain error messages, debug messages, click information, and other unwanted data.
```

Comment:
```suggestion
The resulting visualization may not be exactly what you're looking for. The `message` field is updated with every event, and as a result, it can contain error messages, debug messages, click information, and other unwanted data.
```
?